src/transcode.h

```diff
@@ -144,6 +144,14 @@ transcode_decode_query(struct decode_ctx *ctx, const char *query);
 int
 transcode_encode_query(struct encode_ctx *ctx, const char *query);
 
+/* string representation of the codec
+ *
+ * @in  ctx        decode context
+ * @return         static char* - never NULL, caller does NOT own
```

Comment:
You don't need to specify ownership for const char*. Also please correct "string representation" to "String representation" (or maybe just "Name of the codec"?)